CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to io-sim
+
+We designed `io-classes` to be as close as possible to what `base` package
+provides.  Almost all `IO` instances intstantiate with api provided by one of
```

Comment:
I am not entirely sure what you mean with the second sentence.

Reply:
I wanted to convey they sprit of such insetances as [this one](https://github.com/input-output-hk/io-sim/blob/main/io-classes/src/Control/Monad/Class/MonadSTM.hs?plain=1#L410-L446).